assets/src/edit-story/app/story/useStoryReducer/test/addElements.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Internal dependencies
+ */
+import { setupReducer } from './_utils';
+
+describe( 'addElements', () => {
+	it( 'should add all element to the current page and select them', () => {
```

Comment:
Could we again add a test for adding two elements with the same ID?

Reply:
So, the thing is. No check is actually made for this and it's not one of the invariants maintained by the system (no duplicate ids allowed).

Should it validate this?

Reply:
(I resolved the other similar comment, just FYI)

Reply:
Hmm, okay, sure. Saw that `addElements` is removing duplicates just in case and thought that perhaps testing that could make sense. Would then adding an element twice as a test make more sense?

If you think that's not necessary, that's fine, feel free to ignore :)

Reply:
The tests for `addElementToSelection` reflect that selection should always be unique, and since `addElements` also keeps selection unique, it makes sense to have a test for it IMO.

Reply:
I've changed this since the original comment and added an invariant for no duplicates in selection. I'll add a test as suggested.

Reply:
Oh, I hadn't actually added that invariant, but am doing that now - as it does make a hell of a lot of sense to do 😛.

Reply:
Fixed in 81692ed.